lib/ddtrace/configuration/settings.rb

```diff
@@ -22,6 +23,10 @@ class Settings
               default: -> { env_to_bool(Ext::Runtime::Metrics::ENV_ENABLED, false) },
               lazy: true
 
+      option  :logging_rate,
```

Comment:
Should we bump this under a logging namespace, just like distributed tracing? `Datadog.configuration.logging.rate` seems like it would make a lot of sense.

Reply:
seems reasonable, should I take the same approach we did for distributed tracing? just alias `logging` to `self` for now?

Reply:
Yeah, exactly what I was thinking.